docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -323,11 +359,11 @@ apiVersion: "logging.openshift.io/v1"
 kind: ClusterLogForwarder
 spec:
   pipelines:
-   - name: application-logs
+   - name: app-logs
```

Comment:
nit but there are inconsistencies in indention.  We should on of:

* align arrays with the container element
* indent all arrays same space from the container element